tests/features/licenseexport_cdx.feature

```diff
@@ -0,0 +1,72 @@
+
+Feature: License Explorer
+    As a Platform Eng
+    I want to be able to download the licenses in a csv file format from a specific SBOM
+
+Background:
+    Given User is on TPA Application
```

Comment:
Just a minor thing, but the indentation seems strange. Not sure if that messes with the test code generator, but I would unify it throughout the testplan.